packages/flutter/lib/src/material/scrollbar.dart

```diff
@@ -34,7 +34,61 @@ const Duration _kScrollbarTimeToFade = Duration(milliseconds: 600);
 /// [showTrackOnHover]. The thickness of the track and scrollbar thumb will
 /// become larger when hovering, unless overridden by [hoverThickness].
 ///
-// TODO(Piinks): Add code sample
+/// {@tool dartpad --template=stateless_widget_scaffold}
+///
+/// This sample shows a [Scrollbar] that executes a fade animation as scrolling occurs.
+/// The Scrollbar will fade into view as the user scrolls, and fade out when scrolling stops.
+///
+/// ```dart
+///
+/// Widget build(BuildContext context) {
+///   return Scrollbar(
+///     child: GridView.builder(
+///       itemCount: 120,
+///       gridDelegate:
+///           SliverGridDelegateWithFixedCrossAxisCount(crossAxisCount: 3),
+///       itemBuilder: (BuildContext context, int index) {
+///         return Center(
+///           child: Text('item $index'),
+///         );
+///       },
+///     ),
+///   );
+/// }
+///
+/// ```
+/// {@end-tool}
+/// 
+/// /// {@tool dartpad --template=stateless_widget_scaffold}
+///
+/// This sample shows a [Scrollbar] that executes a fade animation as scrolling occurs.
```

Comment:
```suggestion
/// This sample shows a [Scrollbar] that executes a fade animation as scrolling occurs.
```
It does not execute a fade animation because `isAlwaysShown` is true?